extensions/wikia/SpecialHealthcheck/HealthController.class.php

```diff
@@ -107,47 +107,44 @@ private function occursInArray( $needle, $array ) {
 	}
 
 	/**
-	 * Execute checks for a single database server
-	 * @param string $databaseName Database name to use for connection
+	 * Execute lag check on a given database
 	 * @param LoadBalancer $loadBalancer Load Balancer instance for the given cluster
+	 * @param DatabaseMysqli $db connector to a tested database
 	 * @param int $index Server index to test
-	 * @return bool Is server healthy?
-	 * @throws MWException
+	 * @return bool Is lag below defined limit?
 	 */
-	private function testHost( $databaseName, LoadBalancer $loadBalancer, $index ) {
+	private function testHostLag(LoadBalancer $loadBalancer, DatabaseMysqli $db, $index ) {
```

Comment:
Minor: Code throughout should use the spacey style within the parentheses per our PHP coding conventions.